eng/common/templates/steps/run-build-from-source.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  portableOnly: 'false'
+
+steps:
+  - script: |
+      # Setup AzDo BuildConfig
+      echo "##vso[task.setvariable variable=_BuildConfig]Release"
+    displayName: Setup AzDo BuildConfig
+
+  - script: |
+      set -x
+      df -h
+
+      # Setup build arguments
+      _InternalBuildArgs=""
+      if [[ "$BUILD_REASON" != "PullRequest" && "$SYSTEM_TEAMPROJECT" != "public" ]]; then
+        _InternalBuildArgs="/p:DotNetPublishUsingPipelines=true /p:OfficialBuildId=$(BUILD.BUILDNUMBER)"
+      fi
+      if [[ "${{ parameters.portableOnly }}" == "true" ]]; then
+        _InternalBuildArgs="$_InternalBuildArgs /p:ArcadeBuildPortable=true"
```

Comment:
This logic doesn't make sense to me... "only" being necessary for even one job to be portable...

Reply:
Corrected this.